book/website/collaboration/academic-industry/academic-industry-establish.md

```diff
@@ -76,3 +76,16 @@ Their wide ranging role (from contracting, creating meeting agendas, budgeting,
 There is also a project manager colleague engaged on the Roche side which works very well for ensuring consistency.
 
 Additional resources that may also be of use in these planning stages are the {ref}`Turing Way Project Design Guide <pd-overview>` and the {ref}`Ethical Consideratons when Choosing an Open Source Governance Model chapter <er-ethics-open-source-governance>`. 
+
+### How to Manage Conflict
+Disagreements are inevitable in any collaboration.  The team charter as part of the governance should outline a framework for addressing conflict constructively. While potentially distracting and unpleasant, planning ahead for disagreements can be a prudent way of managing limited resources for the project.
+
+Here are some options to consider:
+* Designated Mediator: Identify a neutral individual within the collaboration or from a third-party organization who can facilitate discussions and mediate any conflicts.
+* Escalation Process: Establish a clear escalation process for addressing disagreements. This could involve initial discussions between directly involved parties, then escalating to team leads or a steering committee if necessary.
```

Comment:
```suggestion
* Escalation process: Establish a clear escalation process for addressing disagreements. This could involve initial discussions between directly involved parties, then escalating to team leads or a steering committee if necessary.
```